source/containers.tex

```diff
@@ -1403,14 +1403,23 @@
 from \tcode{*ranges::begin(rg)}.
 For \tcode{vector},
 if \tcode{R} models
-neither \tcode{ranges::\libconcept{sized_range}} nor \tcode{ranges::\libconcept{forward_range}},
+\tcode{ranges::\libconcept{approximately_sized_range}}
+but not \tcode{ranges::\libconcept{sized_range}} or models
+\tcode{ranges::\libconcept{input_range}}
+but not \tcode{ranges::\libconcept{forward_range}},
 \tcode{T} is also \oldconcept{MoveInsertable} into \tcode{X}.
 
 \pnum
 \effects
 Constructs a sequence container equal to the range \tcode{rg}.
 Each iterator in the range \tcode{rg} is dereferenced exactly once.
 
+\pnum
+\recommended
+If \tcode{R} models \tcode{ranges::\libconcept{approximately_sized_range}} and
+\tcode{ranges::distance(\linebreak{}rg) <= ranges::reserve_hint(rg)} is \tcode{true},
```

Comment:
Why the linebreak here? If we need a break, after `and` should do it

Reply:
The linebreak is to prevent "Overfull"ness. To place the linebreak after the "and" would cause Letex to stretch the line out to the right margin, which would be ugly. What I've done is most consistent with how we've handled similar cases elsewhere.